src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/Clients/InstanceClient.cs

```diff
@@ -240,8 +250,8 @@ public async Task<HttpResponseMessage> PostFileAsAttachment(Instance instance, s
             fileStreamContent.Headers.Add("Content-Disposition", $"attachment; filename={fileName}");
 
             await fileStreamContent.LoadIntoBufferAsync();
-
-            HttpResponseMessage response = await client.PostAsync(requestUri, fileStreamContent);
+            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _validToken);
```

Comment:
In my opinion, it is more readable to put this in the constructor instead of having this line in each method.
Will that cover the needs of the tests?

Reply:
Yeah, I guess all the request use a valid token in this test class so it can be moved to the constructor

Reply:
Changing this for both application and instanceClient